README.md

```diff
@@ -108,6 +106,16 @@ Components in beta stage may include breaking changes, new features, and bug fix
 | [terra-toggle](https://github.com/cerner/terra-core/tree/master/packages/terra-toggle) | [![NPM version](https://badgen.net/npm/v/terra-toggle)](https://www.npmjs.org/package/terra-toggle) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-toggle](https://badgen.net/david/dep/cerner/terra-core/packages/terra-toggle)](https://david-dm.org/cerner/terra-core?path=packages/terra-toggle) |
 | [terra-toggle-button](https://github.com/cerner/terra-core/tree/master/packages/terra-toggle-button) | [![NPM version](https://badgen.net/npm/v/terra-toggle-button)](https://www.npmjs.org/package/terra-toggle-button) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-toggle-button](https://badgen.net/david/dep/cerner/terra-core/packages/terra-toggle-button)](https://david-dm.org/cerner/terra-core?path=packages/terra-toggle-button) |
 | [terra-toggle-section-header](https://github.com/cerner/terra-core/tree/master/packages/terra-toggle-section-header) | [![NPM version](https://badgen.net/npm/v/terra-toggle-section-header)](https://www.npmjs.org/package/terra-toggle-section-header) | ![Stable](https://badgen.net/badge/status/Stable/green) | [![terra-toggle-section-header](https://badgen.net/david/dep/cerner/terra-core/packages/terra-toggle-section-header)](https://david-dm.org/cerner/terra-core?path=packages/terra-toggle-section-header) |
+<!-- AUTO-GENERATED-CONTENT:END *-->
+
+### Deprecated
+
+| Terra Package      | Version | Status | Dependencies |
+|--------------------|---------|--------|--------------|
+| terra-application | [![NPM version](https://badgen.net/npm/v/terra-application)](https://www.npmjs.org/package/terra-application) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey) | ![terra-application](https://badgen.net/badge/dependencies/n%2Fa/grey) |
+| [terra-form](https://github.com/cerner/terra-core/tree/master/packages/terra-form) | [![NPM version](https://badgen.net/npm/v/terra-form)](https://www.npmjs.org/package/terra-form) | ![Deprecated](https://badgen.net/badge/status/Deprecated/grey) | ![terra-form](https://badgen.net/badge/dependencies/n%2Fa/grey) |
```

Comment:
We can unlink the terra-form link at the beginning of this table row.

Reply:
Link removed.